### PR TITLE
dev:Append product service testing logic

### PR DESCRIPTION
**Pull request by kai890707**
**Description**

1. Product service testing is done.
2. Create other services testing file.
3. Other services are on the road.

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] Conforms to style guide

### CODE REVIEW

File: develop/tests/API/v2/OrderTest.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		parent::setUp();
+
+		// Extra code to run before each test
+	}
+
+    public function tearDown(): void
+	{
+		parent::tearDown();
+
+        $this->db->table('db_order')->emptyTable('db_order');
```
Comment (ping-yee) [inline develop/tests/API/v2/OrderTest.php:19]: Do you command `./vendor/bin/php-cs-fixer fix` to these files?
If you really do, then why this line doesn't have indent?

File: develop/tests/API/v2/PaymentTest.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+		parent::setUp();
+
+		// Extra code to run before each test
+	}
+
+    public function tearDown(): void
+	{
+		parent::tearDown();
+
+        $this->db->table('db_payment')->emptyTable('db_payment');
```
Comment (ping-yee) [inline develop/tests/API/v2/PaymentTest.php:19]: Ditto.

**Review by ping-yee — CHANGES_REQUESTED** [review]
Something need to be changed.

**Review by ping-yee — CHANGES_REQUESTED** [review]
Something need to be changed.

File: develop/tests/API/v2/OrderTest.php (hunk 1 above)
Comment (kai890707) [reply develop/tests/API/v2/OrderTest.php:19]: This change is done.

File: develop/tests/API/v2/PaymentTest.php (hunk 2 above)
Comment (kai890707) [reply develop/tests/API/v2/PaymentTest.php:19]: This change is done.

File: develop/tests/API/v2/ProductTest.php [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+     * @test
+     *
+     * update product
+     *
+     * @return void
+     */
+    public function testUpdate()
+    {
+        $productionData = [
+            "name"       => " iphone 21",
```
[the remark this answers is not in the document]
Comment (kai890707) [reply develop/tests/API/v2/ProductTest.php]: This change is done.

File: develop/tests/API/v2/ProductTest.php [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+        $decodeDataExistResults = json_decode($otherDataExistResults->getJSON());
+
+        $decodeDataExistResultsErrMsg = $decodeDataExistResults->messages->error;
+
+        $this->assertEquals($decodeDataExistResultsErrMsg, "Incoming data error");
+
+
+        //success test
+        $data = [
+            "name"  => " iphone 22",
```
[the remark this answers is not in the document]
Comment (kai890707) [reply develop/tests/API/v2/ProductTest.php]: This change is done.

File: develop/tests/API/v2/WalletTest.php [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+		parent::setUp();
+
+		// Extra code to run before each test
+	}
+
+    public function tearDown(): void
+	{
+		parent::tearDown();
+
+        $this->db->table('db_wallet')->emptyTable('db_wallet');
```
[the remark this answers is not in the document]
Comment (kai890707) [reply develop/tests/API/v2/WalletTest.php:19]: This change is done.

**Pull request merged**